src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -187,6 +198,18 @@ public String get(final Object key) {
         return this.attributes.get(key);
     }
 
+    /**
+     * List all values for this key across all appended manifests in the order
+     * they were appended.
+     * @todo: #31 do not expose the internal list structure, or expose readonly
```

Comment:
@pinaf Ok, but that doesn't do the trick. That provides a readonly wrapper around a mutable list. So if I append more, any returned list will appear to be mutated. I will add test. The solution is to return a copy, `return new ArrayList<String>(this.multimap.get(key));`